docs/guides/hooks-list.rst

```diff
@@ -390,6 +390,18 @@ Files
     ``simpletype`` metadata on file entities and make use of it when serving icons and constructing
     ``ege*`` filters and menus.
 
+**upload, file**
```

Comment:
* Validate mime type and prevent upload
 * Change owner / change filestore path/filename